apps/publish-pipeline-result/src/publishResult.ts

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env node
+
+import {
+  CompletedEvent,
+  InProgressEvent,
+  PipelineEvent,
+  PipelineResult,
+  PipelineStatus
+} from "swagger-validation-common/lib/event";
+import * as yargs from "yargs";
+import { logger } from "./logger";
+import { AzureBlobClient } from "./AzureBlobClient";
+import { EventHubProducer } from "./EventHubClient";
+
+type BuildProperties = {
+  buildId: string;
+};
+
```

Comment:
Prefer to use an arrow function. const getBuildProperties = ()=> {buildId: process.env.xxx}

Reply:
good idea, done